auth/src/stub/auth_stub.cc

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Google LLC
```

Comment:
This file should not exist, please put the stub implementations in the Android and Desktop files that already exist.